package.json

```diff
@@ -11,7 +11,7 @@
     "lint:fix": "eslint --fix",
     "test": "npm run test:unit && npm run test:typescript",
     "test:typescript": "tsd",
-    "test:unit": "tap",
+    "test:unit": "node --test --experimental-test-module-mocks",
```

Comment:
the built in mock for node is experimental and we should try to do something like this https://github.com/fastify/fastify-postgres/pull/182#discussion_r1808492921